finch/processes/utils.py

```diff
@@ -286,37 +286,39 @@ def drs_filename(ds: xr.Dataset, variable: str = None):
 def try_opendap(
     input: ComplexInput,
     *,
-    chunks=None,
+    chunks='auto',
```

Comment:
This changes unfortunately breaks analysis on AHCCD stations datasets
```NotImplementedError: Can not use auto rechunking with object dtype. We are unable to estimate the size in bytes of object data```

Reply:
I added a try..except for that case!